torch/testing/_internal/common_distributed.py

```diff
@@ -947,7 +947,7 @@ def threaded_run_test(self):
             self.perThreadTearDown()
 
     def perThreadSetUp(self):
-        pass
+        super().setUp()  # TestCase.setUp() calls torch.manual_seed()
```

Comment:
so is this PR more like syncing the initial seeds between threads? is it still necessary if we do `self.assertEqualRank0`?

Reply:
Yes this PR syncs initial seeds among CPU RNG and GPU RNG on each device. It's not necessary but may help prevent other failure in future.